Wordle.Api/Controllers/WordController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Wordle.Api.Data;
+using Wordle.Api.Dtos;
+using Wordle.Api.Services;
+
+namespace Wordle.Api.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class WordController : ControllerBase
+{
+    private readonly WordService _service;
+
+    public WordController(WordService service)
+    {
+        _service = service;
+    }
+
+    [HttpGet("GetWordsPerPage")]
+    public IEnumerable<Word> Get(int wordsPerPage, int pageNum, string wordFilter)
+    {
+        return _service.GetWordList(wordsPerPage, pageNum, wordFilter);
+    }
+
+    [HttpGet("GetTotalWordCount")]
+    public int Get(string wordFilter)
+    {
+        return _service.GetTotalWordCount(wordFilter);
+    }
+
+    [HttpPost("AddWord")]
+    public void AddWord(string newWord)
+    {
+        _service.AddWord(newWord);
```

Comment:
maybe add a bool return and use that to let the user know it was successful or not.